.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
         runs-on: ubuntu-latest
 
         steps:
-            - uses: softprops/action-gh-release@v2.0.6
```

Comment:
We've don't pin other other actions to exact versions, and this just creates noise from dependabot.